### ISSUE TITLE
Error 500 on user migration

### ISSUE THREAD

**Issue by alicoding** [mozilla/id.webmaker.org#199]
```
Debug: internal, implementation, error
    Error: Error: LoginAPI error
    at Object.exports.create (/Users/ali/projects/id.webmaker.org/node_modules/boom/lib/index.js:21:17)
    at Object.exports.internal (/Users/ali/projects/id.webmaker.org/node_modules/boom/lib/index.js:253:92)
    at Object.exports.badImplementation (/Users/ali/projects/id.webmaker.org/node_modules/boom/lib/index.js:289:23)
    at /Users/ali/projects/id.webmaker.org/web/server.js:379:31
    at /Users/ali/projects/id.webmaker.org/lib/account.js:183:11
    at IncomingMessage.<anonymous> (/Users/ali/projects/id.webmaker.org/lib/account.js:32:16)
    at IncomingMessage.emit (events.js:117:20)
    at _stream_readable.js:944:16
    at process._tickDomainCallback (node.js:486:13)
```

STR:

1. Enter username that never set a password before on `/login`
2. Change the username to username with password set.
3. Press `Set Password`

**Comment by alicoding** [comment]
@cadecairos ^

**Comment by secretrobotron** [comment]
@alicoding fixed by #217?

**Comment by alicoding** [comment]
We handled that client-side now so I'm closing this
